api/AudioBufferSourceNode.json

```diff
@@ -393,7 +393,7 @@
             },
             "opera_android": {
               "version_added": true,
-              "version_removed": "44"
+              "version_removed": "10.1"
```

Comment:
I don't understand this change from 44 to 10.1. Can you explain?

Reply:
I had a bug in the script 🤦🏻‍♂️